### DISCUSSION TITLE
Suggestion for improve the code splitting mechanism

### DISCUSSION THREAD

**Discussion by niklander96** [webpack/webpack#18423] [Q&A]
Hi, everyone)
The diagram illustrates the architecture and request flow:
![image](https://github.com/webpack/webpack/assets/96373865/68186636-ce63-460e-9c78-d930c0fcbcbb)
We need a simple way to separate the JS bundle of the solution into different folders. This way, in the BFF, we can check the requested part of the FE by root: if the request is for the content of a folder, for example, '/secure/..', then we can verify at the BFF level whether the user is authenticated.
This is necessary so that we do not return the entire SPA with potentially sensitive information for unauthenticated users.
Do you plan to enhance the code splitting mechanism to specify the folder where js chunks should be placed?) Do you have any recommendations?)

**Comment by alexander-akait** [comment]
You can write a custom plugin to solve it, I don't think we need an official option for it
